<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateChatroomDialog</class>
 <widget class="QDialog" name="CreateChatroomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="TitleCreate">Create Chatroom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="client.qrc">
    <normaloff>:/icons/red.png</normaloff>:/icons/red.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="CreateChatroomGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string comment="TitleCreate">Create chatroom</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="CreateChatroomNameLabel">
          <property name="text">
           <string comment="ChatName">Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="CreateChatroomDescriptionLabel">
          <property name="text">
           <string comment="Description">Description</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="CreateChatroomNameLineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="CreateChatroomDescriptionTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
